### ISSUE TITLE
[Bug] Edit Bookmark does not display the correct bookmark location.

### ISSUE THREAD

**Issue by 4shutosh** [duckduckgo/Android#1438]
### Describe the bug

Once the user saves a bookmark on a location. Edit Bookmark does not display the correct bookmark location, from the BrowserTabFragment.

### How to Reproduce

1. Open any webpage 
2. Save it as a bookmark in any folder other than the "Bookmarks" root folder
3. After saving edit the bookmark, from the same popup menu.
4. The name being defaulted "Bookmarks" is shown.

### Expected behavior

Precise Bookmark location name should be shown.

### Environment

```markdown
- DDG App Version: develop branch
- Device: Pixel 5
- OS: Android 11
```

**Comment by malmstein** [comment]
@joshliebe can you take a look? I can reproduce this every time.

**Comment by 4shutosh** [comment]
This is going to be fixed in #1441
